src/cmd/manage_nsfs.js

```diff
@@ -388,7 +404,7 @@ async function fetch_existing_account_data(action, target, decrypt_secret_key) {
     let source;
     try {
         const account_path = target.name ?
-            get_config_file_path(accounts_dir_path, target.name) :
+            get_symlink_config_file_path(root_accounts_dir_path, target.name) :
```

Comment:
@alphaprinz @guymguym We need to discuss how to be backward compatible on this whole PR since we want to support online upgrade
we probably need a utility function for that for first trying the new location and fallback to the old location, as it's going to be used in many places

Reply:
I agree :)